stockpile-wrapper/stockpile-wrapper.py

```diff
@@ -73,6 +75,41 @@ def _run_stockpile():
     stdout,stderr = process.communicate()
     return process.returncode
 
+def _check_index(server,port,my_uuid,my_node):
+    _es_connection_string = str(server) + ':' + str(port)
+    es = elasticsearch.Elasticsearch([_es_connection_string],send_get_body_as='POST')
+
+    s = Search(using=es, index="cpuinfo-metadata").query("match", uuid=my_uuid).query("match",node_name=my_node)
+
+    check_results = s.execute()
+
+    # We are using metadata-cpuinfo as it is a basic index that should regularly be there without any extended permissions
+#    check_results = es.search(index="cpuinfo-metadata", body={'query': {'term': {'uuid.keyword': my_uuid, 'node_name.keyword': my_node}}}, size=1)
+    if check_results['hits']['total']['value'] > 0:
+        return True
+    else:
+        return False
+
+def _mark_node(r,my_node,my_uuid,server,port,check_val):
+    current_val = r.get(check_val)
+    print(current_val)
```

Comment:
Did you mean to keep this? Possibly add more output if you want to keep it, so the reader of the log can figure out what this means..

Reply:
I added that for debugging and it will be removed.